llvmlite/tests/test_binding.py

```diff
@@ -1355,6 +1446,28 @@ def test_run(self):
         else:
             raise RuntimeError("expected IR not found")
 
+    def test_run_with_remarks_inline(self):
+        pm = self.pm()
+        pm.add_function_inlining_pass(70)
+        self.pmb().populate(pm)
+        mod = self.module(asm_inlineasm2)
+        (status, remarks) = pm.run_with_remarks(mod)
+        self.assertTrue(status)
+        # Inlining has happened?  The remark will tell us.
+        self.assertIn("Passed", remarks)
+        self.assertIn("inlineme", remarks)
+
+    def test_run_with_remarks_inline2(self):
+        pm = self.pm()
+        pm.add_function_inlining_pass(0)
+        self.pmb().populate(pm)
+        mod = self.module(asm_inlineasm3)
+        (status, remarks) = pm.run_with_remarks(mod)
+        self.assertTrue(status)
+        # Inlining has happened?  The remark will tell us.
+        self.assertIn("Passed", remarks)
+        self.assertIn("inlineme", remarks)
```

Comment:
I think it would be good to adjust these tests to assert through the remarks that in one case the function inlines and in another it does not (this should be forceable through attributes `noinline` and `alwaysinline`).